docs/sdks/client-side-identity.md

```diff
@@ -431,28 +447,28 @@ Returns the current stored identity, if available.
 
 If there is a valid identity available, the return value is an object representing the full stored identity. The properties of the object are the same as the stored value as described in the [contents structure](#contents-structure) section.
 
-If there is no currently valid identity (even if the identity is only temporarily unavailable), the return value will be null. If you need to know whether the identity is only temporarily unavailable, you can call [isLoginRequired()](#isloginrequired-boolean).
+If there is no currently valid identity (even if the identity is only temporarily unavailable), the return value is null. If you need to know whether the identity is only temporarily unavailable, you can call [isLoginRequired()](#isloginrequired-boolean).
 
 ## UID2 Storage Format
 
 The SDK uses either local storage or a first-party cookie to store the user's identity. The default option is to use local storage, but this can be changed using an [init parameter](#init-parameters).
 
-Even when using local storage, the SDK will check to see if there is a newer identity available in a first-party cookie. This will allow the SDK to make use of local storage while still allowing you to provide an identity by setting the first-party cookie.
+Even when using local storage, the SDK checks to see if there is a newer identity available in a first-party cookie. This allows the SDK to make use of local storage while still allowing you to provide an identity by setting the first-party cookie.
 
-### Properties
+### UID2 Cookie Properties
 
-If cookie storage is being used, the cookie will use the properties in this table.
+If cookie storage is being used, the cookie uses the properties in the following table.
 
 | Properties | Default Value | Comments |
 | :--- | :--- | :--- |
 | `Name` | `__uid_2` | N/A |
-| `Expiry` | N/A | The value is the refresh token expiration timestamp as specified in the [POST /token/generate](../endpoints/post-token-generate.md) or [POST /token/refresh](../endpoints/post-token-refresh.md) response. |
+| `Expiry` | N/A | The value is the refresh token expiration timestamp as specified in the<br/>[POST /token/generate](../endpoints/post-token-generate.md) or [POST /token/refresh](../endpoints/post-token-refresh.md) response. |
```

Comment:
Is the `<br/>` intentional here? It can put the line break in an unfortunate place at certain browser widths, e.g.:
![image](https://github.com/IABTechLab/uid2docs/assets/111951647/6b319887-d662-4153-9bbf-dca989cfa0a3)

Reply:
Thx! Removed it. Yes it was intentional, and it looked good to me. My bad.